### ISSUE TITLE
tensorflow-1.4.1-cp36-cp36m-macosx_10_13_x86_64.whl report Illegal instruction: 4

### ISSUE THREAD

**Issue by smoothdvd** [lakshayg/tensorflow-build-archived#20]
When I running demo of model/research/slim,

scripts/finetune_inception_resnet_v2_on_flowers.sh: line 73: 90776 Illegal instruction: 4  python train_image_classifier.py --train_dir=${TRAIN_DIR} --dataset_name=flowers --dataset_split_name=train --dataset_dir=${DATASET_DIR} --model_name=${MODEL_NAME} --checkpoint_path=${PRETRAINED_CHECKPOINT_DIR}/${MODEL_NAME}.ckpt --checkpoint_exclude_scopes=InceptionResnetV2/Logits,InceptionResnetV2/AuxLogits --trainable_scopes=InceptionResnetV2/Logits,InceptionResnetV2/AuxLogits --max_number_of_steps=1000 --batch_size=32 --learning_rate=0.01 --learning_rate_decay_type=fixed --save_interval_secs=60 --save_summaries_secs=60 --log_every_n_steps=10 --optimizer=rmsprop --weight_decay=0.00004

**Comment by lakshayg** [comment]
@frankhinek can you please look into this issue?

**Comment by frankhinek** [comment]
Given the limited information about the environment when this error was encountered, I can't reproduce it.

The only hint I would have is to ensure that your local CPU supports all of the same instruction sets (SSE4.1, SSE4.2, AVX, AVX2, FMA).

Beyond that, I would suggest that if there are issues of this nature that the questions be directed to the providers of the code or the TensorFlow team.  The builds of TensorFlow in this repo are simply compiled versions of the official releases.

**Comment by smoothdvd** [comment]
@frankhinek machdep.cpu.leaf7_features: SMEP ERMS RDWRFSGS TSC_THREAD_OFFSET BMI1 HLE AVX2 BMI2 INVPCID RTM FPU_CSDS
iMac (Retina 5K, 27-inch, Late 2014)

**Comment by frankhinek** [comment]
@smoothdvd Does a TF 1.4.1 with Python 3.5 build work on your machine?

**Comment by smoothdvd** [comment]
@frankhinek Official wheel of TF 1.4.1 with Python 3.5 works

**Comment by frankhinek** [comment]
The best suggestion I can share is to compile a version with 1.4.1 for your system.  There must be some difference in the instruction sets your CPU uses.

This is why the official TensorFlow project releases the non-optimized versions of the CPU wheels.  It is really impossible to know all the various combinations that exist out there.

If you do build one, feel free to upload to this repo and tag which features are enabled.

**Comment by Ghost-Zhou** [comment]
I successfully apply the tensorflow-1.4.0-cp36-cp36m-macosx_10_12_x86_64.whl into my computer, and I also keep the new version of tensorflow(1.5.0), I hope I could build and upload the new version for you, but I could only build on python 2.7, I don't know why.

**Comment by lakshayg** [comment]
Closing this issue since it is related to a very old version of tensorflow. Please feel free to reopen in case the problem persists with newer binaries.
